modules/ROOT/pages/audit-logging.adoc

```diff
@@ -1138,6 +1138,41 @@ Environments that are created automatically when you create an organization or b
 
 |===
 
+== Anypoint DataGraph
+
+[%header,cols="25a,15a,15a,15a,15a,30a"]
+|===
+|User Action
+|Object Type
+|Object
+|Parent
+|Action
+|Payload
+
+| Add API
+| User
+| data-graph-{api-name}-{env-name}
+| N/A
+| Create
```

Comment:
Add API (not Create)

Reply:
![image](https://user-images.githubusercontent.com/39770499/116116695-4b9c1400-a670-11eb-9d8b-132f61ec7d40.png)